canonical/charmhub.io
# Get last modified date for interfaces

## Done
- get the last modified date for each interface
## How to QA
- Go to https://charmhub-io-1550.demos.haus/interfaces/ingress.json
- check that response has `last-modified`
## Issue / Card WD-380
Fixes #

## Screenshots
